safeeyes/safeeyes/config/lang/es.json

```diff
@@ -0,0 +1,42 @@
+{
+    "meta_info": {
+        "language_name": "Español",
+        "language_name_en": "Spanish"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Vete a andar un poco",
+            "Reclínate sobre tu silla y relájate"
+        ],
+        "short_break_exercises": [
+            "Cierra bien tus ojos",
+            "Mueve tus ojos en círculos",
```

Comment:
Does this provide the same meaning as "Move your eyes left to right and vice versa?"